query/influxql/function.go

```diff
@@ -134,6 +134,21 @@ func createFunctionCursor(t *transpilerState, call *influxql.Call, in cursor, no
 		}
 		cur.value = value
 		cur.exclude = map[influxql.Expr]struct{}{call.Args[0]: {}}
+	case "cumulative_sum":
+		value, ok := in.Value(call.Args[0])
```

Comment:
What does `value` represent?

Reply:
This value becomes the `functionCursor` value. I'm not super sure what its utility is, but it was required for the implementation of functionCursor further down in the file. From lines 398 onward.